lib/signet/oauth_2/client.rb

```diff
@@ -883,7 +883,7 @@ def to_jwt(options={})
       # @note A serialized client contains sensitive information. Persist or transmit with care.
       #
       # @return [String] A serialized JSON representation of the client.
-      def to_json
+      def to_json(options={})
```

Comment:
It looks like the `json` library allows arbitrary numbers of arguments. Maybe better:

```ruby
def to_json(*)
```

Reply:
Yes thanks that works. Updating the PR.